### PR TITLE
Update README to point at DevSite docs

### PR DESCRIPTION
**Pull request by mcloonan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ easy-to-use client library for the [Google Ads API][].
 
 # Documentation for Users
 
+**For thorough documentation of all features and utilities, visit our
+[Developers Site ruby guide][].**
+
```
Comment (jradcliff) [inline README.md]: Lines 12-13 duplicate line 14

File: README.md [hunk 2]
```diff
@@ -8,6 +8,11 @@ easy-to-use client library for the [Google Ads API][].
...
+[Developers Site ruby guide]: https://developers.google.com/google-ads/api/docs/client-libs/ruby/
+
 ## Getting started
 
 `google-ads-googleads` will allow you to connect to the
```
Comment (jradcliff) [inline README.md]: GitHub won't let me comment on unchanged lines, but I noticed the wording of "In order to achieve so" on line 16 is a bit awkward. Could you rephrase that?

File: README.md [hunk 3]
```diff
@@ -8,6 +8,11 @@ easy-to-use client library for the [Google Ads API][].
 
 # Documentation for Users
 
+**For thorough documentation of all features and utilities, visit our
+[Developers Site ruby guide][].**
```
Comment (jradcliff) [inline README.md]: Site -> site
ruby -> Ruby (I assume)

File: README.md [hunk 4]
```diff
@@ -59,7 +81,7 @@ with no arguments:
 Alternatively, if you prefer to keep the file elsewhere, you can instantiate the
 library by passing the path to where you keep this file:
```
Comment (jradcliff) [inline README.md]: Instead of library, do you mean client?

File: README.md (hunk 1 above)
Comment (mcloonan) [reply README.md]: No, line 14 defines where the link goes. I'm using a different type of markdown link here than we've used in other places.

File: README.md (hunk 2 above)
Comment (mcloonan) [reply README.md]: Removed that whole sentence. It didn't seem to add much.

Comment (jradcliff) [reply README.md]: Sorry, I wrote that comment before I realized the link style you're using.
